modules/custom/az_migration/migrations/migrate_plus.migration_group.az_migration.yml

```diff
@@ -0,0 +1,25 @@
+# The machine name of the group, by which it is referenced in individual migrations.
+id: az_migration
```

Comment:
This file might be able to remain in `config/install` since it's a migration group, not a migration.

Reply:
Sounds good. I will make that change and commit